src/lang/translation_provider_CPUFLASH.hpp

```diff
@@ -53,7 +53,7 @@ class CPUFLASHTranslationProviderBase : public ITranslationProvider {
     }
 
     // 2020-08-18 updated to 340 buckets with more texts coming in. With previous 256 buckets, collisions occured already (we have 240+ texts)
-    using SHashTable = string_hash_table<hash_djb2, buckets_count, 256>; ///< beware of low numbers of buckets - collisions may occur unexpectedly
+    using SHashTable = string_hash_table<hash_djb2, buckets_count, 310>; ///< beware of low numbers of buckets - collisions may occur unexpectedly
```

Comment:
comment say 340 ant number is 310

Reply:
the comment was referring to the number of buckets (not texts), but it became obsolete anyway, I'll fix it